docs/csharp/misc/cs0119.md

```diff
@@ -19,18 +19,20 @@ ms.assetid: 048924f1-378f-4021-bd20-299d3218f810
 - A method identifier is used as if it were a struct or class  
   
 ## Example  
- The following sample generates CS0119.  
-  
-```csharp  
-// CS0119.cs  
-using System;  
-public class MyClass
-{  
-   public static void Test() {}  
-  
-   public static void Main()  
-   {  
-      Console.WriteLine(Test.x);   // CS0119  
-   }  
+ The following sample generates CS0119: 'C.B()' is a method, which is not valid in the given context.
```

Comment:
The important addition is how a developer can fix this error. I'd add the following:

```suggestion
 The following sample generates CS0119: 'C.B()' is a method, which is not valid in the given context. You can fix this error by changing the name of the method `C.B`, or using the fully qualified name for the class `B`.
```

To make this explanation more clear, it would be helpful to put `B` (the class) in a namespace like `N2`. Then, you can say "...for the class `N2.B`."